case_studies/psf_variation/train_models.sh

```diff
@@ -1,27 +1,20 @@
 #!/bin/bash
 
-# Runs the experiments for the BLISS PSF variation paper.
+# This script runs the experiments from the BLISS spatially-variant PSF paper.
 
-# train base model
-bliss -cp ~/bliss/case_studies/psf_variation -cn config mode=train \
-    train.trainer.logger.name=PSF_MODELS \
-    train.trainer.logger.version=base_model \
-    encoder.image_normalizer.concat_psf_params=False \
-    cached_simulator.cached_data_path=/data/scratch/aakash/train_single_94-1-12
+# Generate new data
+bash ~/bliss/scripts/generate_data_in_parallel.sh -n 32 -cp ~/bliss/case_studies/psf_variation/conf -cn psf_aware
+bash ~/bliss/scripts/generate_data_in_parallel.sh -n 32 -cp ~/bliss/case_studies/psf_variation/conf -cn single_field
```

Comment:
probably good to rename the script since it includes data generation now

Reply:
(which is great, thanks for the change. now it's fully reproducible)

Reply:
Renamed to `run_experiments.sh`